lib/menu.rb

```diff
@@ -0,0 +1,53 @@
+require_relative 'dish'
+require_relative 'order'
+require_relative 'sms'
+
+class Menu
+
+  require 'csv'
+
+  attr_reader :current_order, :list
+
+  def initialize
+    @list = []
+    load_menu
+  end
+
+  def view_menu
```

Comment:
Have a think about whether the details of menu printing code belong in the Order class.